### PR TITLE
VpnServerRouteForm

### PR DESCRIPTION
**Pull request by isaaceichelberger**
- did the thing

### CODE REVIEW

**Review by jvallexm — APPROVED** [review]

File: storybook/src/stories/Forms/VpnServerRouteForm.stories.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    },
+    invalidTextCallback: {
+      description:
+        "A function that returns a string of invalid text to display when the `name` field is invalid",
+      type: { required: true }, // required prop or not
+      control: "none",
+    },
+    isModal: {
+      description: "A boolean describing whether the form is in a modal or not",
+      type: { required: true },
```
Comment (evpirnia) [inline storybook/src/stories/Forms/VpnServerRouteForm.stories.js:44]: isModal is not required according to props

File: src/components/forms/VpnServerRouteForm.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        />
+      </IcseFormGroup>
+    );
+  }
+}
+
+export default VpnServerRouteForm;
+
+VpnServerRouteForm.defaultProps = {
+  data: { name: "", destination: "", action: "translate" },
```
Comment (evpirnia) [inline src/components/forms/VpnServerRouteForm.js:79]: action here is lowercase but story description for action states 
```
        "A string value of the action to be taken, can be `Translate`, `Deliver`, or `Drop`",
```

is this case sensitive?

**Review by evpirnia — CHANGES_REQUESTED** [review]

File: src/components/forms/VpnServerRouteForm.js (hunk 2 above)
Comment (isaaceichelberger) [reply src/components/forms/VpnServerRouteForm.js:79]: yes omg good catch

File: storybook/src/stories/Forms/VpnServerRouteForm.stories.js [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+  }
+
+  function invalidTextCallback(stateData, componentProps) {
+    return contains(["foo", "bar"], stateData.name)
+      ? `Key name ${stateData.name} already in use.`
+      : `Invalid Key Name. Must match the regular expression: /^[A-z]([a-z0-9-]*[a-z0-9])?$/i`;
+  }
+  return (
+    <VpnServerRouteForm
+      data={{ name: "route", destination: "2.2.2.2/16", action: "deliver" }}
```
[the remark this answers is not in the document]
Comment (isaaceichelberger) [reply storybook/src/stories/Forms/VpnServerRouteForm.stories.js:78]: yea it shouldve been lowercase in the story

**Review by evpirnia — APPROVED** [review]

**Pull request merged**